test/test_array_ops.cpp

```diff
@@ -316,6 +316,55 @@ TEST(ArrayOps, ExpandDims) {
 
 }  // end of test op ExpandDims
 
+// Test op:GatherND
+TEST(ArrayOps, GatherNd_2D) {
+  int dim1 = 2;
+  int dim2 = 2;
+
+  Tensor indices(DT_INT32, TensorShape({2, 1}));
+  AssignInputValues<int>(indices, {0, 1});
+
+  Tensor params(DT_FLOAT, TensorShape({dim1, dim2}));
+  AssignInputValuesRandom(params);
+
+  vector<int> static_input_indexes = {};
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  Scope root = Scope::NewRootScope();
+  auto R = ops::GatherNd(root, params, indices);
+  std::vector<Output> sess_run_fetchoutputs = {R};
+
+  OpExecuter opexecuter(root, "GatherNd", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+
+}  // end of test op GatherND
+
+// Test op:GatherND
+TEST(ArrayOps, GatherNd_3D) {
```

Comment:
Please remove underscore from gtest names. apparently gtest reserves the right to use underscores willy-nilly